x-pack/platform/plugins/shared/alerting/server/saved_objects/model_versions/rule_model_versions.ts

```diff
@@ -42,4 +43,11 @@ export const ruleModelVersions: SavedObjectsModelVersionMap = {
       create: rawRuleSchemaV4,
     },
   },
+  '5': {
+    changes: [],
+    schemas: {
+      forwardCompatibility: rawRuleSchemaV4.extends({}, { unknowns: 'ignore' }),
```

Comment:
Should this be rawRuleSchemaV5.extends?

Reply:
Oops good catch! I fixed in [this](https://github.com/elastic/kibana/pull/216292/commits/5cefd1ca4a069612e50a2cb594f526a62f813fd6) commit